# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/libwebrtc/audio/audio_receive_stream.cc
# Commit: d0b311007c03
# Full Hash: d0b311007c033e83824f5f6996a70ab9e870f31f
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Get RTCP BYE and RTP timeout handling working again (from Bug 1595479) r=mjf,dminor
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106145
# ==============================================================================

diff -r daa9eac709dc -r d0b311007c03 third_party/libwebrtc/audio/audio_receive_stream.cc
--- a/third_party/libwebrtc/audio/audio_receive_stream.cc	Fri Feb 19 16:20:38 2021 +0000
+++ b/third_party/libwebrtc/audio/audio_receive_stream.cc	Fri Feb 19 15:56:41 2021 -0600
@@ -46,6 +46,8 @@
     }
   }
   ss << ']';
+  ss << ", rtcp_event_observer: "
+     << (rtcp_event_observer ? "(rtcp_event_observer)" : "nullptr");
   ss << '}';
   return ss.str();
 }
@@ -83,7 +85,7 @@
       config.jitter_buffer_min_delay_ms,
       config.jitter_buffer_enable_rtx_handling, config.decoder_factory,
       config.codec_pair_id, config.frame_decryptor, config.crypto_options,
-      std::move(config.frame_transformer));
+      std::move(config.frame_transformer), config.rtp.rtcp_event_observer);
 }
 }  // namespace
 